Makefile

```diff
@@ -23,20 +23,20 @@ get:
 	go mod tidy && go mod download
 
 get-lint:
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.46.2
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.50.1
 
 .PHONY: lint
 lint:
 	if [ ! -f ./bin/golangci-lint ]; then \
 		$(MAKE) get-lint; \
 	fi;
-	./bin/golangci-lint run ./... --timeout 5m0s
+	./bin/golangci-lint run ./... --timeout 5m0s -v
```

Comment:
What is `-v`?

Reply:
`-v` means `verbose` to provide linting debug info